src/tools/concurrency/ActorExecutionTrace.java

```diff
@@ -153,15 +156,15 @@ public static synchronized void returnBuffer(final ByteBuffer b) {
     PromiseCreation((byte) 2, 17),
     PromiseResolution((byte) 3, 28),
     PromiseChained((byte) 4, 17),
-    Mailbox((byte) 5, 17),
+    Mailbox((byte) 5, 21),
 
     // at the beginning of buffer, allows to track what was created/executed
     // on which thread, really cheap solution, timestamp?
     Thread((byte) 6, 9),
 
     // for memory events another buffer is needed
     // (the gc callback is on Thread[Service Thread,9,system])
-    MailboxContd((byte) 7, 19),
+    MailboxContd((byte) 7, 23),
```

Comment:
those number are used to determine if the buffer has enough free space for the event